queries-postgres-uptime.yml

```diff
@@ -0,0 +1,6 @@
+pg_postmaster_uptime:
+  query: "select extract(epoch from current_timestamp - pg_postmaster_start_time()) as seconds"
```

Comment:
why epoch? why not seconds?

Reply:
"Seconds" shows only amount of seconds now. But it's required to get the number of seconds since 1970-01-01 00:00:00-00

Reply:
ok, got it, than it's expected behaviour. Even if it looks weird :)